infra/templates/gitlab-bootstrap-iac-cicd/README.md

```diff
@@ -0,0 +1,196 @@
+# Bootstrap for Terraform deployments through Gitlab into Azure
+
+This repository is intended to simplify setting up CICD required for the infrastructure CICD on the Gitlab platform. It aims to do the following:
```

Comment:
```suggestion
This repository is intended to simplify setting up the Azure components that are needed for the Gitlab platform to perform Azure infrastructure CICD automation. It aims to do the following:
```
Relationship between Azure and Gitlab was not immediately understood by me.

Reply:
I think the right documentation is a mix of what you have here and some other context that I did not include. I'll re-phrase, thanks for calling this out!

Reply:
@TechnicallyWilliams what do you think about the latest phrasing?

Reply:
wow. Super clear. Love it.